### PR TITLE
Fix a bug in FENE interaction

### PR DESCRIPTION
**Pull request by jsmrek**

### CODE REVIEW

File: src/interaction/FENE.hpp [hunk 1]
```diff
@@ -97,13 +104,11 @@ namespace espressopp {
 
         real ffactor;
         
-        if(r0 == 0) {
+        if(r0 != 0) {
```
Comment (junghans) [inline src/interaction/FENE.hpp]: Uh, ugly! A comparison `r>A_SMALL_NUMBER` would be better!

**Comment by junghans** [comment]
And please update copyright in `src/interaction/FENE.hpp`.

**Comment by govarguz** [comment]
Besides the comments already made by @junghans and if @stuehn has no additional ones. It looks OK to me

**Comment by MrTheodor** [comment]
Please check if the documentation is correct. 
Would it be possible for you to write some sort of test to check the correctness (in testsuiet/) ?

**Comment by jsmrek** [comment]
@junghans Ok, I'll do the changes, what small number would you suggest?
@MrTheodor I'm not quite sure, what documentation you are referring to. 
It makes more sense to me if someone else would check it against her\his calculations. Of course, I checked myself the correctness for a bunch of parameters by plotting forces and energies and comparing with analytical results.

**Comment by MrTheodor** [comment]
@jsmrek I mean the one in .hpp file that goes to developer doc and the one in doc/ug/;

**Comment by niktre** [comment]
@MrTheodor Do not want that you get me wrong, but since when do we actually care about developer doc? 

@jsmrek use `real threshold_ = 1e-30`. I think it is enough.

For user documentation that @MrTheodor mentions, it would be great if you provide some description of the FENE potential in the python file. As a quick reference, look at the `LennardJones93Wall.py` (lines 23 till 43). There you'll find the formula (written in `.. math:` environment), and a couple of usage examples. If you are not sure about your documentation in python-file (whether it makes sense, for example :)), check your email. 

Do not work at the weekend! Remember, you have a couple of girls to take care about!!

**Comment by MrTheodor** [comment]
@niktre I don't know, but it exists so why not to keep it updated or remove it; IMHO
@jsmrek there is a definition of very small number in `src/include/esconfig.hpp` `ROUND_ERROR_PREC`.

**Comment by niktre** [comment]
It was a long way from a simple bug to an acceptable documentation! 👍

**Pull request merged**